modin/pandas/test/test_groupby.py

```diff
@@ -15,7 +15,8 @@
 import pandas
 import numpy as np
 import modin.pandas as pd
-from modin.pandas.utils import from_pandas, to_pandas, try_cast_to_pandas
+from modin.utils import try_cast_to_pandas
+from modin.pandas.utils import from_pandas, to_pandas
```

Comment:
Why `try_cast_to_pandas` is imported from  `modin.utils`, but `to_pandas` is imported from `modin.pandas.utils`?

Reply:
`to_pandas` was also moved to `modin.utils`